### PR TITLE
BD-2956: API usage dashboard EA - MERGE 4/2

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

Closes #**BD-2956**

#### Is this change associated with a Braze feature/product release?
- [X] Yes (**April 1, 2024**)
- [ ] No

---

### CODE REVIEW

**Review by marisak — APPROVED** [review]
While you didn't ask for my review, I wanted to say that this looks good to me!

File: _docs/_hidden/private_betas/api_usage_dashboard.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+{% alert note %}
+The **Last 3 Hours** and **Last 6 Hours** options will show traffic by minutes. Larger time periods will show traffic by hours or days.
+{% endalert %}
+
+## Considerations
+
+The API usage dashboard includes all REST API requests that Braze received and returned a `2XX`, `4XX`, or `5XX` response for. This includes Data Transformation outputs and Cloud Data Ingestion syncs. SDK traffic and User Update steps are not included in this dashboard.
+
+During periods of high usage, you can refresh the dashboard up to 4 times per minute. You may need to wait a few minutes before refreshing the dashboard again.
```
Comment (bartlam18) [inline _docs/_hidden/private_betas/api_usage_dashboard.md:68]: Can we add a line here:

"Data shown in the dashboard may have up to a 15 minute delay in showing recent traffic."

File: _docs/_hidden/private_betas/api_usage_dashboard.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+- **Success rate:** The percentage of total requests where Braze issued a `2XX` success response.
+- **Error rate:** The percentage of total requests where Braze issued a `4XX` or `5XX` error response.
+
+## Dashboard controls
+
+![]({% image_buster /assets/img/api_usage_dashboard/filters.png %}){: style="float:right;max-width:35%;margin-left:15px;"}
+
+### Filters
+
+The dashboard can be filtered to show a more narrow view of REST API traffic for your workspace. These filters include:
```
Comment (bre-fitzgerald) [inline _docs/_hidden/private_betas/api_usage_dashboard.md:29]:
```suggestion
Apply filters to narrow the view of REST API traffic for your workspace. Available filters include:
```

File: _docs/_hidden/private_betas/api_usage_dashboard.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+- API endpoint
+- API key
+- Response code
+
+### Group data
+
+The dashboard can be configured to group data into various data series. The grouping options are:
```
Comment (bre-fitzgerald) [inline _docs/_hidden/private_betas/api_usage_dashboard.md:37]:
```suggestion
Group data into various data series to explore different patterns in your usage. Available grouping options are:
```

File: _docs/_hidden/private_betas/api_usage_dashboard.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+- Response codes (default)
+- API endpoint
+- API key
+- Success and failure only
+
+### Date
+
+The dashboard can be configured to show a smaller or greater time range. The available date options include:
```
Comment (bre-fitzgerald) [inline _docs/_hidden/private_betas/api_usage_dashboard.md:46]:
```suggestion
Adjust the date filter to show a smaller or greater time range as needed. Available date options include:
```

**Review by bre-fitzgerald — APPROVED** [review]
@lydia-xie lgtm! Left some suggestions for adjusting the filters language to be more user-focused than product-focused, apply or reject at your discretion 😄

**Pull request merged**